server/api.js

```diff
@@ -16,4 +16,22 @@ router.get("/clients", (req, res) => {
 	});
 });
 
+router.post("/clients", (req, res) => {
+
+	const newClientDateAdded = req.body.date_added;
+	const newClientName = req.body.name;
+	const newClientBikes = req.body.bikes_needed;
+	const newClientPhoneNumber = req.body.phone_number;
```

Comment:
You could also validate these values, but for a prototype I think it's fine for now.